homeassistant/components/skybell/__init__.py

```diff
@@ -65,8 +65,11 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
         )
 
     # Clean up unused cache file since we are using an account specific name
-    if os.path.exists(hass.config.path(DEFAULT_CACHEDB)):
-        os.remove(hass.config.path(DEFAULT_CACHEDB))
+    # Remove with import
+    if await hass.async_add_executor_job(
+        os.path.exists, hass.config.path(DEFAULT_CACHEDB)
+    ):
+        await hass.async_add_executor_job(os.remove, hass.config.path(DEFAULT_CACHEDB))
```

Comment:
Please avoid creating multiple executor job, each one has to be pushed into the queue, picked up, run and returned.  If you create a single function to call and only do one call to the executor it is much more efficient.